scripts/oci_integration_tests.sh

```diff
@@ -116,8 +121,12 @@ for case in "${test_cases[@]}"; do
   mkdir -p "$(dirname $logfile)"
   sudo RUST_BACKTRACE=1 RUNTIME=${RUNTIME} ${OCI_TEST_DIR}/validation/$case >$logfile 2>&1 || (cat $logfile && exit 1)
   if [ 0 -ne $(grep "not ok" $logfile | wc -l ) ]; then
-      cat $logfile
-      exit 1
+    if [ 0 -eq $(grep "# cgroupv2 is not supported yet " $logfile | wc -l ) ]; then
+      echo "Skip $case bacause oci-runtime-tools doesn't support cgroup v2"
+      continue;
+    fi
+    cat $logfile
+    exit 1
```

Comment:
Hey, I'm not sure I understand the purpose of this part. If we are checking for the test in `check_environment` function above, why do we need this here? Also, shouldn't the `0 -eq ...` part be `0 -ne ...` instead? If the grep finds occurrence, then wc will output a non-zero number, in which case we should be entering the condition, right?

Reply:
The logic is that if the test fails and the cause of the failure is not supported by the test in cgroup v2, the failure is ignored. I could have used check_environment, but I added this because of the uniform error messages.